### PR TITLE
ci(coverage): fix coverage by upgrading truffle

### PR DESCRIPTION
**Pull request by mrice32**
**Motivation**

The nightly coverage run is broken.

**Summary**

Upgraded the truffle version to be fully node14 compatible.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [x]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

**Issue(s)**

N/A

### CODE REVIEW

**Review by chrismaree — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
     "pretty-quick": "^2.0.1",
     "secp256k1": "^3.7.1",
     "solidity-docgen": "^0.5.3",
-    "truffle": "^5.1.33"
+    "truffle": "^5.1.66"
```
Comment (nicholaspai) [inline package.json:55]: Big upgrade!

**Review by daywiss — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (mrice32) [reply package.json:55]: Just 33 patches :)

**Comment by pemulis** [comment]
@mrice32 is this PR still active?

**Comment by Gamaranto** [comment]
What's the status on this? @mrice32

**Comment by mrice32** [comment]
@pemulis @Gamaranto I kept meaning to come back to this, but @nicholaspai superseded this with a more complete PR here: https://github.com/UMAprotocol/protocol/pull/2695. So I think this is safe to close.

**Pull request closed without merging**